docs/configuration.md

```diff
@@ -118,6 +118,7 @@ storage:
     secretkey: awssecretkey
     region: us-west-1
     regionendpoint: http://myobjects.local
+    s3accelerate: false
```

Comment:
```suggestion
   accelerate: false
```

I suggest removing the `s3` prefix. This config is under the S3 driver section, so the prefix is redundant.